### PR TITLE
LPS-103653 Remove portal-kernel-*-compat

### PR DESCRIPTION
**Pull request by dantewang**
@tinatian See https://github.com/brianchandotcom/liferay-portal/pull/80013#issue-332353636

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:help

**Comment by dantewang** [comment]
ci:help

**Comment by dantewang** [comment]
ci:test

Liferay Continuous Integration <notifications@github.com> 于 2019年10月30日周三
17:23写道：

**Comment by dantewang** [comment]
ci:test

File: modules/modules.properties [hunk 1]
```diff
@@ -196,14 +196,11 @@ bundle.version[com.liferay.asset.categories.test]=1.0.0
 bundle.version[com.liferay.asset.category.property.api]=3.0.0
 bundle.version[com.liferay.asset.category.property.service]=3.0.0
 bundle.version[com.liferay.asset.category.property.test]=1.0.0
-bundle.version[com.liferay.asset.display.api]=3.0.0
```
Comment (tinatian) [inline modules/modules.properties:199]: @dantewang 
Some of the changes are not related to this pull. Could you please make it clear what should be updated with your change? Please resend.

Thanks.
Tina.

**Comment by tinatian** [comment]
Hi @dantewang ,
There is an SF failure, please also fix it and resend.

Thanks.
Tina
